src/apis/folder-apis/transferFolder.ts

```
@@ -0,0 +1,16 @@
+
+export async function transferFolder(
+  data: TransferFolderData
+): Promise<TransferFolderResponse> {
+  try {
+    const response = await axiosInstance.post(
+      '/api/dispatch/directory-transmissions',
+      data
+    );
+    return response.data;
+  } catch (error) {
+    throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)